server/db/models/campaign.js

```diff
@@ -23,13 +23,9 @@ class Campaign extends BaseModel {
         },
         type: { type: 'string', enum: ['Starter', 'Accelerator', 'Grant'] },
         page_url: { type: 'string', minLength: 1 },
-        logo_url: { type: 'string', minLength: 1 },
-        letters_goal: {
-          anyOf: [{ type: 'integer', minimum: 0 }, { type: 'null' }]
-        },
-        donation_goal: {
-          anyOf: [{ type: 'integer', minimum: 0 }, { type: 'null' }]
-        }
+        campaign_tagline: { type: 'string' },
+        campaign_text: { type: 'string', minLength: 1 },
```

Comment:
I think omitting the `minLength` like @paramsiddharth said is the right move. Let's do it on `campaign_text` also and this should be ready to merge.

Reply:
Sorry for the delayed reply on this thread. If it's optional you can handle that by setting a default value or making it a nullable field.